app/tree/test/models/graphql/GraphQlSchemaDefinitionSpec.scala

```diff
@@ -97,16 +103,18 @@ class GraphQlSchemaDefinitionSpec extends PlaySpec {
       )))
       result must be(Json.parse(
         s"""
-           |{"data":{"getBoroughsByCity":[${TestData.city.boroughs.map(borough => "{\"uri\":" + "\"" + borough + "\"").mkString("},")}}]}}
+           |{"data":{"boroughs":{"byCity":[${TestData.city.boroughs.map(borough => "{\"uri\":" + "\"" + borough + "\"").mkString("},")}}]}}}
            |""".stripMargin
       ))
     }
     "return a geometry given a city" in {
       val query =
         graphql"""
            query TreesQuery($$city: CityFieldsInput!) {
-              getGeometryOfCity(city: $$city) {
-                wkt
+              cities{
+                geometryOfCity(city: $$city) {
```

Comment:
This would actually be better on the root query, city($$city) { geometry }, as opposed to cities { ... }. Also, can you distinguish URIs (cityUri) from domain models (city) in the naming? Why does geometry-of-city need more than a URI?

Reply:
This is a dud function. Never got rid of it my b